<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CalibrationValues</class>
 <widget class="QWidget" name="CalibrationValues">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>496</width>
    <height>334</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Calibration</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="6" column="0">
    <widget class="QLineEdit" name="grat6ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>6</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_2">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Offset</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLineEdit" name="grat9ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>9</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="off1ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="text">
      <string>Grating</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLineEdit" name="grat7ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>7</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="5" column="0">
    <widget class="QLineEdit" name="grat5ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>5</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLineEdit" name="grat3ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>3</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="QLineEdit" name="gad1ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>98000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="label_3">
     <property name="font">
      <font>
       <pointsize>12</pointsize>
       <weight>75</weight>
       <bold>true</bold>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>Gadjust</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLineEdit" name="grat4ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>4</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLineEdit" name="grat8ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>8</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLineEdit" name="grat2ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>2</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="grat1ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>1</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QLineEdit" name="off2ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>1536000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QLineEdit" name="off3ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>3072000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="off5ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>1536000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="off7ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QLineEdit" name="off6ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>3072000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QLineEdit" name="off8ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>1536000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLineEdit" name="off9ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>3072000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="4">
    <widget class="QLineEdit" name="gad2ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>98000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="4">
    <widget class="QLineEdit" name="gad3ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>98000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="4">
    <widget class="QLineEdit" name="gad4ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>98000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="4">
    <widget class="QLineEdit" name="gad5ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>98000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QLineEdit" name="off4ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>0</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="6" column="4">
    <widget class="QLineEdit" name="gad6ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>98000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="8" column="4">
    <widget class="QLineEdit" name="gad8ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>98000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="4">
    <widget class="QLineEdit" name="gad7ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>98000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="4">
    <widget class="QLineEdit" name="gad9ET">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="text">
      <string>98000</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="5">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="6">
    <widget class="QPushButton" name="PrintButton">
     <property name="text">
      <string>Print</string>
     </property>
    </widget>
   </item>
   <item row="2" column="6">
    <widget class="QPushButton" name="SaveButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="3" column="6">
    <widget class="QPushButton" name="RestoreButton">
     <property name="text">
      <string>Restore</string>
     </property>
    </widget>
   </item>
   <item row="4" column="6">
    <widget class="QPushButton" name="ExitButton">
     <property name="text">
      <string>Exit</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
